open_event/views/admin/home.py

```diff
@@ -6,14 +6,30 @@
 from flask_admin import expose
 from flask_admin.base import AdminIndexView
 from flask.ext.scrypt import generate_password_hash
+from datetime import datetime
+import jwt
 
 from ...helpers.data import DataManager, save_to_db,get_google_auth,get_facebook_auth
 from ...helpers.data_getter import DataGetter
 from ...helpers.helpers import send_email_after_account_create, send_email_with_reset_password_hash
-from open_event.models.user import User
 from open_event.helpers.oauth import OAuth, FbOAuth
 from flask import current_app
 
+def generate_token(user):
```

Comment:
@aviaryan But we cannot make the form directly post to the API endpoint ... JWT expects a json request with email and password. Form does not give json. We could make form give json by using ajax, but that would further complicate the login. And we also need to access Flask-login and create a logged in session for the user. Doing that from the API would not be right.